### PR TITLE
Note that framed should also be enabled in docs.

### LINKED ISSUE #96: Thrift compact protocoll should also be framed.
In [the docs](http://frontera.readthedocs.org/en/latest/topics/frontera-settings.html#hbase-use-compact-protocol) you do not mention that one should also use the framed protocol.

### PR DESCRIPTION
**Pull request by lljrsr**
Fixes #96 .

### CODE REVIEW

**Comment by sibiryakov** [comment]
should we also rename setting parameter?

**Comment by lljrsr** [comment]
I think that's a good idea.

File: docs/source/topics/frontera-settings.rst [hunk 1]
```diff
@@ -505,15 +505,15 @@ Default: ``9090``
...
-HBASE_USE_COMPACT_PROTOCOL
+HBASE_USE_FRAMED_COMPACT
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Default: ``False``
 
-Whatever workers should use Thrift compact protocol. Dramatically reduces transmission overhead, but needs to be turned
-on on server too.
+When enabled workers will use TFramedTransport and TCompactProtocol features of Thrift. Dramatically reduces transmission overhead, 
+but the server needs to be configured to use framed transport and compact protocol.
```
Comment (sibiryakov) [inline docs/source/topics/frontera-settings.rst]: It seems a broken a english to me :) What about
> Enabling this option dramatically reduces transmission overhead, but the server needs to be ...

**Comment by sibiryakov** [comment]
could you please rebase it to master again.

**Comment by lljrsr** [comment]
Okay :ear_of_rice:

**Comment by sibiryakov** [comment]
:beers: Thank you!

**Pull request merged**